### PR TITLE
feat: interactive Bubble Tea TUI and awake time accuracy fix

### PR DESCRIPTION
**Pull request by alexinslc**
## Summary

- **Interactive TUI**: Running `rekap` now launches a full-screen Bubble Tea TUI with sidebar + detail pane layout (k9s-style). Navigate sections with j/k, drill into expanded views with Enter, go back with Esc, quit with q.
- **Awake time fix**: Uptime collector now parses `pmset -g log` for sleep/wake events and subtracts sleep periods from awake time, giving accurate "time actually at the machine" instead of wall-clock time since midnight.
- **`--print` flag**: Old static dump behavior preserved via `rekap --print`. Also auto-detects non-TTY (piping) and falls back to static output.
- **Demo TUI**: `rekap demo` also launches the TUI with fake data. Use `rekap demo --print` for static.

### TUI Sections
| Section | Summary View | Expanded View (Enter) |
|---|---|---|
| System | Uptime, battery, screen time | Boot time, lock count, plug events |
| Productivity | Top 3 apps, focus streak | All 10 apps, switching stats, bundle IDs |
| Browser | Tabs, top domain, URLs | Per-browser breakdown, work/distraction stats |
| Network | Interface, bytes up/down | Network name, since-boot qualifier |
| Wellness | Fragmentation score, warnings | Score breakdown with weights, all burnout checks |
| Media | Track + app | Same |
| Notifications | Total, top app | Full top apps list |
| Issues | Count | Full issue list with trackers |

### Key Decisions
- TUI is the new default when stdout is a TTY
- `--print`, `--json`, `--quiet` are mutually exclusive
- Unavailable sections shown grayed out with "(n/a)"
- Title bar shows date, footer shows key hints
- User's configured theme/colors applied to TUI
- Minimum 80 columns; below that shows "terminal too small"

## Testing
- 8 unit tests for sleep/wake event parsing (all edge cases)
- `go test ./...` passes
- `go vet ./...` clean
- Manual testing: `--print`, `--json`, `--quiet`, pipe detection, mutual exclusion

## Post-Deploy Monitoring & Validation
No additional operational monitoring required: CLI tool with no server component. Users test locally.

---

[Compound Engineered](https://github.com/EveryInc/compound-engineering-plugin) Generated with [Claude Code](https://claude.com/claude-code)

Co-Authored-By: Claude Opus 4.6 <noreply@anthropic.com>

### CODE REVIEW

File: internal/collectors/uptime.go [hunk 1]
```diff
@@ -74,3 +82,65 @@ func CollectUptime(ctx context.Context) UptimeResult {
 	result.Available = true
 	return result
 }
+
+var sleepPattern = regexp.MustCompile(`\bSleep\b`)
+var wakePattern = regexp.MustCompile(`\bWake\b`)
+
+// collectSleepDuration runs pmset -g log and returns total sleep time between start and end.
+func collectSleepDuration(ctx context.Context, start, end time.Time) time.Duration {
+	cmd := exec.CommandContext(ctx, "bash", "-c", "pmset -g log 2>/dev/null | grep -E '(Sleep|Wake)\\b'")
```
[the remark this answers is not in the document]
Comment (alexinslc) [reply internal/collectors/uptime.go]: Fixed. Now calls `exec.CommandContext(ctx, "pmset", "-g", "log")` directly and filters Sleep/Wake lines in Go, removing the shell pipeline dependency.

File: internal/ui/tui/model.go [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+func applyThemeColors(cfg *config.Config) {
+	if cfg == nil {
+		return
+	}
+	if cfg.Accessibility.Enabled && cfg.Accessibility.HighContrast {
+		primaryColor = lipgloss.Color("15")
+		secondaryColor = lipgloss.Color("15")
+		accentColor = lipgloss.Color("15")
+		successColor = lipgloss.Color("15")
```
[the remark this answers is not in the document]
Comment (alexinslc) [reply internal/ui/tui/model.go]: Fixed. Introduced a `colorPalette` struct that lives on the Model. `colorsFromConfig()` returns a palette, and `buildStylesFromPalette()` builds styles from it. No more global color mutation.

File: internal/collectors/uptime.go [hunk 3]
```diff
@@ -74,3 +82,65 @@ func CollectUptime(ctx context.Context) UptimeResult {
...
+	output, err := cmd.Output()
+	if err != nil {
+		return 0
+	}
+	return parseSleepWakeEvents(string(output), start, end)
+}
+
+// parseSleepWakeEvents parses filtered pmset log output and returns total sleep duration
+// between start and end times. Exported for testing.
```
[the remark this answers is not in the document]
Comment (alexinslc) [reply internal/collectors/uptime.go]: Fixed. Comment now reads: "Internal helper, tested via same-package tests."

File: internal/collectors/uptime.go [hunk 4]
```diff
@@ -74,3 +82,65 @@ func CollectUptime(ctx context.Context) UptimeResult {
...
+	var totalSleep time.Duration
+	var sleepStart time.Time
+	inSleep := false
+
+	scanner := bufio.NewScanner(strings.NewReader(output))
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		tsMatches := timestampPattern.FindStringSubmatch(line)
+		if len(tsMatches) < 2 {
```
[the remark this answers is not in the document]
Comment (alexinslc) [reply internal/collectors/uptime.go]: Fixed: Added local uptimeTimestampPattern in uptime.go to eliminate cross-file dependency on battery.go's timestampPattern. Commit: 031eca1

File: internal/collectors/uptime.go [hunk 5]
```diff
@@ -74,3 +82,65 @@ func CollectUptime(ctx context.Context) UptimeResult {
...
+			continue
+		}
+		if !strings.HasPrefix(tsMatches[1], today) {
+			continue
+		}
```
[the remark this answers is not in the document]
Comment (alexinslc) [reply internal/collectors/uptime.go]: Fixed: Now allows yesterday's date for cross-midnight sleep intervals. Intervals are clamped to [start, end]. Added 3 cross-midnight test cases. Commit: 031eca1
